Documentation/developer-guide.md

```diff
@@ -23,58 +23,61 @@ Please read [Contributing](https://github.com/dotnet/wcf/blob/master/Documentati
 Building the repository
 =======================
 
-The WCF repository can be built from a regular, non-admin command prompt using build.cmd. This build produces a single System.Private.ServiceModel assembly that implements the individual client libraries.
+The WCF repository can be built from a regular, non-admin command prompt using build.cmd on Windows or build.sh on *nix.
```

Comment:
NIT: consider using "non-Windows" or "other" platforms, not sure if people would associate OS X as "*nix"